src/RA_Dlg_GameTitle.cpp

```diff
@@ -108,7 +108,7 @@ INT_PTR Dlg_GameTitle::GameTitleProc(HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM
                     {
                         //	Existing title
                         ASSERT(m_aGameTitles.find(std::string(sSelectedTitle)) != m_aGameTitles.end());
-                        nGameID = m_aGameTitles[std::string(Narrow(sSelectedTitle))];
+                        nGameID = m_aGameTitles[std::string(ra::Narrow(sSelectedTitle))];
```

Comment:
i realize the code was doing this before, but `Narrow` already returns a `std::string`, it's unnecessary to wrap it again.

Reply:
Sounds good to me.